api-urls-regex.js

```diff
@@ -0,0 +1,9 @@
+// URL and regex which matches the IP in the response.
```

Comment:
I would call the file `shared.js` and use a named export instead.

This also needs to be added to `files` in package.json.